### PR TITLE
Lookup route link by network uuid

### PR DESCRIPTION
**Pull request by akry**

### CODE REVIEW

File: vnet/lib/vnet/endpoints/1.0/responses/translation_static_address.rb [hunk 1]
```diff
@@ -10,6 +10,12 @@ def self.generate(object)
 
         route_link = object.batch.route_link.commit
         res[:route_link_uuid] = route_link.uuid if route_link
+
+        i_network = object.batch.ingress_network.commit
+        e_network = object.batch.egress_network.commit
+
+        res[:ingress_network_uuid] = i_network.uuid if i_network
+        res[:egress_network_uuid] = e_network.uuid if e_network
```
Comment (rakshasa) [inline vnet/lib/vnet/endpoints/1.0/responses/translation_static_address.rb]: This ends up calling vnmgr twice. Should instead check if object.foo_network_id is valid, and then do batch.foo_network.canonical_uuid.commit.

File: vnet/lib/vnet/models/route_link.rb [hunk 2]
```diff
@@ -22,5 +22,24 @@ class RouteLink < Base
...
+      RouteLink.dataset.join_table(
+        :inner,
+        :routes,
+        route_links__id: :routes__route_link_id
+      ).join_table(
+        :inner,
+        :networks,
+        routes__network_id: :networks__id
+      ).where(:networks__uuid => uuid.gsub("nw-","")).select(:route_links__uuid).all.map(&:uuid)
+    end
```
Comment (rakshasa) [inline vnet/lib/vnet/models/route_link.rb]: https://github.com/axsh/openvnet/blob/develop/vnet/lib/vnet/models/ip_lease.rb#L57

The above is more like how it should look.

**Comment by rakshasa** [comment]
+1

**Comment by Metallion** [comment]
Is this code still required? If so, can you solve the merge conflicts so the CI can be run again? Also since this code changes the API and the database, could you write a short description of what problem is being solved here? I can't immediately figure it out just by looking at the code.

**Comment by akry** [comment]
we don't need this code. just delete the branch.

**Pull request closed without merging**